upstream/CVE-2018-1002100.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "CVE-2018-1002100",
+  "modified": "2018-03-16T19:24:46Z",
+  "published": "2018-03-16T19:24:46Z",
+  "summary": "Kubectl copy doesn't check for paths outside of it's destination directory.",
+  "details": "In Kubernetes versions 1.5.x, 1.6.x, 1.7.x, 1.8.x, and prior to version 1.9.6, the kubectl cp command insecurely handles tar data returned from the container, and can be caused to overwrite arbitrary local files.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubectl"
```

Comment:
Description says about `k8s.io/kubernetes`.
Also `kubectl` doesn't have these tags.

Updated in https://github.com/aquasecurity/vuln-list-k8s/pull/60/commits/6e6e4a9bd1df8737b60de8567fbdd1635399437e

Reply:
Isn't it a bug in `kubectl`?

Reply:
I looked into the fix. It is more like a client issue than a k8s cluster issue.
https://github.com/kubernetes/kubernetes/pull/61298/files

Reply:
I'm a little confused. Thanks for this note. Now I have a better understanding of how to find a vulnerable package.

Reply:
revert in https://github.com/aquasecurity/vuln-list-k8s/pull/60/commits/02095b7158a766cbb015d152790ceb65201be3e2